napari/util/temporary_file.py

```diff
@@ -0,0 +1,26 @@
+from tempfile import NamedTemporaryFile
+from contextlib import contextmanager
+import os
+
+
+@contextmanager
+def temporary_file(suffix=''):
+    """Yield a writeable temporary filename that is deleted on context exit.
```

Comment:
```suggestion
    """Yield a writeable temporary filename that is deleted on context exit.

```

Reply:
also the correct spelling is "writable" ;)

Reply:
done